.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Style/Encoding:
+  Enabled: false
+  
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 140
+
+Metrics/BlockLength:
+  Max: 55
```

Comment:
don't use so many symbols for block

don't use more than 100-120 symbols for line length